.github/workflows/dev.yml

```diff
@@ -19,5 +19,7 @@ jobs:
     uses: axonivy-market/github-workflows/.github/workflows/ci.yml@v2
```

Comment:
Hi @ivy-rew 
Should we change the `ci.yml` uses with a new tag name? e.g. v3?
When we merge it into the master branch, do we run `git tag` commands manually or have a job to do the tagged task?

Reply:
you can decide on this, though I'd suggest to keep the 'v2' tag for now, as I think there is no functional/breaking change in it and we we can 'spare' the roll-out effort to update all the consumers to the new version.

Once the thing is on the master branch you can tag it, either using the github-ui or with git commands in your terminal. There is no automated job for this.

Reply:
Will you take the lead on merging this and tagging it @phhung-axonivy ? Or shall I apply it?
Or do we have further need for discussion?

Reply:
You introduced two new inputs in the ci.yml file, and you used them in the dev.yml file.
With ci.yml@v2, these inputs have not been introduced yet. That's the reason why I ask you to use a new tag v3.
If it's taking too much of your time, please apply it.

Reply:
Ok I'll introduce it and consume it by example in msgraph and a-trust ... so you can use this as copy example how to proceed in the future

Reply:
Thank you so much @ivy-rew

Reply:
v3 is introduced: the adaption can be copied from here ... I think you can share this way to go with the team and apply it to all artifacts in the market.

- a-trust: https://github.com/axonivy-market/a-trust-connector/pull/30